src/node_worker.cc

```diff
@@ -349,6 +349,8 @@ void Worker::Run() {
             static_cast<EnvironmentFlags::Flags>(environment_flags_),
             thread_id_,
             std::move(inspector_parent_handle_)));
+        // TODO(addaleax): Adjust for the embedder API snapshot support changes
+        env_->set_builtin_loader(env()->builtin_loader());
```

Comment:
I think this is only safe when shared-RO heap is enabled, but here that's not part of the build config yet, so maybe a TODO?

Reply:
With the updates here, I’ve hidden these calls behind the shared-RO heap flag, but I’d be curious to know more about why this is otherwise unsafe.